src/watcher.js

```diff
@@ -195,7 +198,46 @@ class Watcher extends SafeEventEmitter {
             })
         );
 
+        let twitchSendMessage;
+
+        const emitSendMessage = sendState => this.emit('chat.send_message', sendState);
+
+        function bttvSendMessage() {
+            try {
+                let defaultPrevented = false;
+                const sendState = {
```

Comment:
this should be a message send class that gets instantiated on message send, don't ya think?